### PR TITLE
Adds --skip-bundle option to skip javascript bundling

### PR DESCRIPTION
**Pull request by alexandervalencia**
Hey @robert-frampton,<br><br>Attached is the fix for [Issue #78](https://github.com/electricjs/electric/issues/78).<br><br>Let me know if you have any questions.<br><br>Thanks!

### CODE REVIEW

File: lib/commands/run.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = {
 	run: function(options) {
 		if (configExists()) {
 			runTask('build', ['watch', 'server'], {
+				skipBundle = options.skipBundle,
```
Comment (robframpton) [inline lib/commands/run.js]: Did this not throw a syntax error when you tested? You can't set object literal properties this way.

**Review by robframpton — CHANGES_REQUESTED** [review]

**Comment by robframpton** [comment]
Hey @alexandervalencia, can you make the change regarding my comment above? Thanks

**Comment by robframpton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alexandervalencia** [comment]
@Robert-Frampton Corrections pushed. Thanks for catching that.

**Comment by robframpton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**